src/ceph.in

```diff
@@ -849,7 +849,11 @@ def main():
         return 1
 
     if parsed_args.help:
-        return do_extended_help(parser, childargs)
+        return do_extended_help(parser, childargs, ('mon', ''), ' '.join(childargs))
+
+    # implement "tell module help"
+    if len(childargs) >= 3 and childargs[0] == 'tell' and childargs[2] == 'help':
+        return do_extended_help(parser, childargs, childargs[1].split('.'), None)
```

Comment:
could you read my comment again ? and try `ceph tell osd help` and `ceph tell mon help` ? if they don't throw, i am fine.